internal/daysteps/daysteps.go

```diff
@@ -12,9 +19,53 @@ const (
 )
 
 func parsePackage(data string) (int, time.Duration, error) {
-	// TODO: реализовать функцию
+	parts := strings.Split(data, ",")
+	if len(parts) != 2 {
+		return 0, 0, errors.New("неверный формат данных")
+	}
+
+	steps, err := strconv.Atoi(parts[0])
+	if err != nil {
+		return 0, 0, err
```

Comment:
Лучше оборачивать ошибку, чтобы понимать где она произошла
```suggestion
		return 0, 0, fmt.Errorf("conversion error: %w", err)
```